Workbooks/AKS/Smart Insights/Smart Detectors.workbook

```diff
@@ -61,7 +61,7 @@
             "crossComponentResources": [
               "value::selected"
             ],
-            "value": "/subscriptions/692aea0b-2d89-4e7e-ae30-fffe40782ee2/resourceGroups/jobyaks2/providers/Microsoft.OperationalInsights/workspaces/jobyaks2",
+            "value": "",
```

Comment:
you should remove the value field entirely.  leaving it like this may sets the value to "explicitly nothing selected", instead of letting the value be automatically set by a query, etc.

Reply:
Thanks for the suggestion, fixed